cmake/modules/FindPulseAudio.cmake

```diff
@@ -21,12 +21,16 @@ endif()
 if(PKG_CONFIG_FOUND)
   pkg_check_modules(PC_PULSEAUDIO libpulse>=${PulseAudio_FIND_VERSION} QUIET)
   pkg_check_modules(PC_PULSEAUDIO_MAINLOOP libpulse-mainloop-glib QUIET)
+  pkg_check_modules(PC_PULSEAUDIO_SIMPLE libpulse-simple QUIET)
 endif()
 
-find_path(PULSEAUDIO_INCLUDE_DIR NAMES pulse/pulseaudio.h
+find_path(PULSEAUDIO_INCLUDE_DIR NAMES pulse/pulseaudio.h pulse/simple.h
                                  PATHS ${PC_PULSEAUDIO_INCLUDEDIR} ${PC_PULSEAUDIO_INCLUDE_DIRS})
 
-find_library(PULSEAUDIO_LIBRARY NAMES pulse libpulse
+find_library(PULSEAUDIO_LIBRARY NAMES pulse libpulse pulse-simple libpulse-simple
```

Comment:
undo this please, it is searched for in the check you added below.
otherwise looks good

Reply:
Thanks much - removed it and squashed it in!